as2/6.3/modules/ROOT/pages/as2-connector-example-receive-send.adoc

```diff
@@ -0,0 +1,285 @@
+		<logger level="INFO" doc:name="Message Received!" message="Message Received: #[payload]"/>
+	</flow>
+</mule>
+----
+
+[[synchronous-mdns]]
+== Sending Synchronous AS2 MDNs
+
+Creating this example involves creating a new Mule project and configuring the *HTTP>Listener* source and the *Logger* component.
+
+The following screenshot shows the Anypoint Studio app flow for this example:
+
+image::as2-connector-example-sync-flow.png[Send Synchronous AS2 MDNs Example App Flow]
+
+=== Create a Flow to Send Synchronous AS2 MDNs
+
+Configure the *HTTP>Listener* source to initiate a Mule flow when a call is made to the `+/sendSyncFlow+` path on localhost port `8082`:
+
+. In the *Mule Palette* view, select the *HTTP > Listener* source and drag it on to the canvas. 
+. On the *HTTP > Listener* properties screen, optionally change the value of the *Display Name* field and click *OK*.
+. Set the *Path* field to `+/sendSyncFlow+`.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the *HTTP > Listener* source in the app.
+. On the *General* tab, accept the defaults and click *OK*.
+
+=== Add the Send with Sync MDN Operation
+
+Add the *Send with Sync MDN* operation and configure it to send MDNs to the receiving endpoint of `partnera`:
+
+. Drag the *Send with Sync MDN* next to *Listener* on the canvas.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element for the *Send with Sync MDN* operation.
+. On the *General* tab, configure the connection by setting the *Partner URL* field to `+http://localhost:8082/as2-receive+`.
++
+. Specify the self and partner credentials.
++
+For the self configuration, configure these fields in the *Self Config* section:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|AS2 Partner Name | `partnerb`  
+|x509 Alias | `parnerb`
+|email | `+support@partnerb.com+`
+|===
++
+For the partner configuration, configure these fields in the *Partner Config* section:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|AS2 Partner Name | `partnera` 
+|x509 Alias | `parnera`
+|email | `+support@partnera.com+`
+|===
++
+. Optionally configure the *Requester Config* section with the algorithms to use.
+. In the *Keystore* section, configure the keystore that contains the certificates and keys:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Keystore Password| `+test+` 
+|Keystore Path | `+as2/partnerb.p12+`
+|Private Key Password | `test`
+|===
+
+=== Add the Logger Component
+
+Add the *Logger* component to the flow and configure it to log a message to the Studio console 
+when the app sends a synchronous MDN.
+
+. From the *Mule Palette* view, select *Core* and drag the *Logger* component next to *Send with Sync mDN*.
+. In the Logger properties screen, set the *Message* field to `Message Sent and MDN Received!`.
+
+=== Save and Test the Mule App
+
```

Comment:
Do we need an introductory sentence before the steps?

Reply:
No, the first paragraph introduces the steps. I got this approach from Kim's edits.